x-pack/plugins/snapshot_restore/public/app/services/http/use_request.ts

```diff
@@ -88,8 +96,5 @@ export const useRequest = ({ path, method, body, interval }: UseRequest) => {
     loading,
     data,
     request,
-    setIsMounted: (status: boolean) => {
```

Comment:
With the addition of the cleanup function return by `useEffect` call, I don't think is necessary.